process_report/invoices/pi_specific_invoice.py

```diff
@@ -43,31 +70,109 @@ def _prepare(self):
+                ],
+                axis=1,
+            )
+
+        # Add a row containing sums for certain columns
+        column_sums = list()
+        sum_columns_list = list()
+        for column_name in self.TOTAL_COLUMN_LIST:
+            if column_name in pi_projects.columns:
+                column_sums.append(pi_projects[column_name].sum())
+                sum_columns_list.append(column_name)
+        pi_projects.loc[
+            len(pi_projects)
+        ] = None  # Adds a new row to end of dataframe initialized with None
+        pi_projects.loc[pi_projects.index[-1], invoice.INVOICE_DATE_FIELD] = "Total"
+        pi_projects.loc[pi_projects.index[-1], sum_columns_list] = column_sums
+
+        # Add dollar sign to certain columns
+        for column_name in self.DOLLAR_COLUMN_LIST:
+            if column_name in pi_projects.columns:
+                pi_projects[column_name] = pi_projects[column_name].apply(
+                    add_dollar_sign
+                )
+
+        pi_projects.fillna("", inplace=True)
+
+        return pi_projects
+
     def export(self):
-        def _export_pi_invoice(pi):
-            if pandas.isna(pi):
-                return
-            pi_projects = self.export_data[self.export_data[invoice.PI_FIELD] == pi]
-            pi_instituition = pi_projects[invoice.INSTITUTION_FIELD].iat[0]
-            pi_projects.to_csv(
-                f"{self.name}/{pi_instituition}_{pi} {self.invoice_month}.csv"
+        def _create_html_invoice(temp_fd):
+            environment = Environment(loader=FileSystemLoader(TEMPLATE_DIR_PATH))
+            template = environment.get_template("pi_invoice.html")
+            content = template.render(
+                data=pi_dataframe,
+            )
+            temp_fd.write(content)
+            temp_fd.flush()
+
+        def _create_pdf_invoice(temp_fd_name):
+            chrome_binary_location = os.environ.get(
+                "CHROME_BIN_PATH", "usr/bin/chromium"
+            )
+            if not os.path.exists(chrome_binary_location):
+                sys.exit(
+                    f"Chrome binary does not exist at {chrome_binary_location}. Make sure the env var CHROME_BIN_PATH is set correctly or that Google Chrome is installed"
+                )
+
+            invoice_pdf_path = (
+                f"{self.name}/{pi_instituition}_{pi}_{self.invoice_month}.pdf"
+            )
+            subprocess.run(
+                [
+                    chrome_binary_location,
+                    "--headless",
+                    "--no-sandbox",
+                    f"--print-to-pdf={invoice_pdf_path}",
+                    "--no-pdf-header-footer",
+                    "file://" + temp_fd_name,
+                ],
+                capture_output=True,
             )
 
         self._filter_columns()
+
         if not os.path.exists(
             self.name
         ):  # self.name is name of folder storing invoices
             os.mkdir(self.name)
```

Comment:
You can simplify this using `os.makedirs`:

```suggestion
        os.makedirs(self.name, exist_ok=True)
```